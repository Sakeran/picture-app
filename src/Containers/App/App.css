.App {
  margin-top: 5em;
}

.App-main {
  max-width: 900px;
  margin: 0 auto;
}
