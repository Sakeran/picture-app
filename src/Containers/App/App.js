import React, { Component } from 'react';
import { BrowserRouter as Router, Route, Switch} from 'react-router-dom';
import 'normalize.css';
import './App.css';

import HeaderContainer from '../../Containers/HeaderContainer/HeaderContainer';
import Index from '../../Pages/Index/Index';
import Login from '../../Pages/Login/Login';
import NotFound from '../../Pages/NotFound/NotFound';
import Signup from '../../Pages/Signup/Signup';

class App extends Component {
  render() {
    return (
      <Router>
        <div className="App">
          <HeaderContainer />
          <div className="App-main">
            <Switch>
              <Route exact path="/" component={Index} />
              <Route exact path="/login" component={Login} />
              <Route exact path="/signup" component={Signup} />
              <Route component={NotFound} />
            </Switch>
          </div>
        </div>
      </Router>
    );
  }
}

export default App;
