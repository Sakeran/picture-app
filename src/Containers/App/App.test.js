import React from 'react';
import { App } from './App';
import { shallow } from 'enzyme';
import toJson from 'enzyme-to-json';

it('renders without crashing', () => {
  const component = shallow(<App />);
  const tree = toJson(component);
  expect(tree).toMatchSnapshot();
});
