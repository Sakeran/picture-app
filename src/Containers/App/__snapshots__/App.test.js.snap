// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders without crashing 1`] = `
<BrowserRouter>
  <div
    className="App"
  >
    <Connect(HeaderContainer) />
    <div
      className="App-main"
    >
      <Routes />
    </div>
  </div>
</BrowserRouter>
`;
